VocaDbWeb.Resources/Resources/Views/Shared/EntryCreateStrings.resx

```diff
@@ -1,147 +1,150 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What happened here, why was the file overwritten?

Reply:
@Pyther99 Please convert line endings to CRLF. The .editorconfig isn't working properly?

Reply:
@ycanardeau I am using CRLF 
![image](https://user-images.githubusercontent.com/65015656/112977479-65703700-9156-11eb-9b0f-ecee0a9785d9.png)

Reply:
Seems like the original file was saved as LF.

Reply:
Okay, the initial commit was probably with LF. That was many years ago though, most files were already converted, but guess this one wasn't. It's probably ok then.